src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -46,7 +46,7 @@ class HeaderUserMenu extends React.Component {
                 ''
               )}
               <p>
-                <strong>{user.plan.remainingCredits}</strong>
+                <strong>{user.plan.remainingCredits}</strong>{' '}
```

Comment:
👍

Reply:
In the update of this PR, Prettier removed that space and put everything in the same line; works too, but wanted to tell you the differente, @andresmoschini

Reply:
Well, if it works and it is _pretty_, it is fine for me. ;)